exercises/php/cards/Suit.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace McrDigital\PhpFundamentals1\Cards;
+
+class Suit
+{
+    public array $suit = [
```

Comment:
Maybe this could be a static list or even an enum?  Google suggests they're a thing in PHP but I'm no expert.